content/posts/0.20-release.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Pyodide 0.12.0 release"
+date: 2022-01-10
```

Comment:
Need updating once we know the release date.

Reply:
Aiming for today =)